docroot/themes/custom/uids_base/scss/global.scss

```diff
@@ -441,6 +446,32 @@ body:not(.page-node-type-article, .page-node-type-page).layout-builder-disabled
   }
 }
 
+
+
+// Section w/ color background padding
+
+div[class*="uids-component--"] {
+  .lb__container {
```

Comment:
I moved this to target the`lb__container` because we were trying to add all padding to this container instead of the parent container.  Let me know if you see the parent container as being used for padding elsewhere.